src/10_functions.py

```diff
@@ -9,4 +14,8 @@
 # Print out "Even!" if the number is even. Otherwise print "Odd"
 
 # YOUR CODE HERE
+if (num % 2 == 0) :
```

Comment:
this would be a good place to try ternary syntax if you like.

Reply:
Made it work with ternaries 👍